### ISSUE TITLE
Hydrants not Being Selected 

### ISSUE THREAD

**Issue by latitude35** [Esri/utilities-isolation-trace#7]
I've successfully ran the trace widget with our water system data. I can select a water line and the widget will return valves and customers in the results of the trace. For some reason the isolation trace has never been able to return hydrants. I've selected water line segments that have hydrants off of them but they are never included in the isolation trace results. 

Hydrants are points and a simple junction feature type in our water system. Is there anything we would need to tweak in our water system data or geometric network so a hydrant is selected in a trace when applicable?

**Comment by MikeMillerGIS** [comment]
Do you have hydrant valves in the same valve layer as your isolation/system valves.  If so, they need to be removed.

**Comment by latitude35** [comment]
The hydrants are in a separate feature class but the hydrant valves are in the same feature class as the system valves. However, I put a definition query on the system valves so the included hydrant valves would not be drawn. They need to be removed, as in deleted from the system valves and made into their own feature class?

**Comment by MikeMillerGIS** [comment]
Correct, the geometric trace only lets you set a feature class as a disable layer, not a subset of those features

**Comment by latitude35** [comment]
Okay thanks. I'll remove the hydrant valves.
